atomicfu-gradle-plugin/src/main/kotlin/kotlinx/atomicfu/plugin/gradle/AtomicFUGradlePlugin.kt

```diff
@@ -33,10 +33,7 @@ open class AtomicFUGradlePlugin : Plugin<Project> {
         val pluginVersion = rootProject.buildscript.configurations.findByName("classpath")
             ?.allDependencies?.find { it.name == "atomicfu-gradle-plugin" }?.version
         extensions.add(EXTENSION_NAME, AtomicFUPluginExtension(pluginVersion))
-        if (config.transformJs && config.jsVariant == "IR") {
-            // apply the compiler plugin, if IR transformation should be applied to KotlinJsIrTarget compilations
-            plugins.apply(AtomicfuKotlinGradleSubplugin::class.java)
-        }
+        plugins.apply(AtomicfuKotlinGradleSubplugin::class.java)
```

Comment:
I’ve figured, that we can only obtain values passed to the `atomic {}` extension after project evaluation, though `jsVariant` option was supposed to define whether to apply the compiler plugin (`AtomicfuKotlinGradleSubplugin`) and compiler plugin should be applied before project evaluation is completed. 

For now, compiler plugin is applied by default.

Maybe there is some other way to create a user flag, that can be resolved before the project is evaluated?